examples/hellogrpc/e2e-test.sh

```diff
@@ -88,6 +109,18 @@ delete() {
   bazel-bin/examples/hellogrpc/${LANGUAGE}/server/staging.delete
 }
 
+# e2e test that checks that args are added to the kubectl apply command
+check_kubectl_args() {
+    # Checks that bazel run <some target> does pick up the args attr and
+    # passes it to the execution of the template
+    EXPECT_CONTAINS "$(bazel run examples/hellogrpc:staging.apply)" "apply --v=2"
+    # Checks that bazel run <some target> -- <some extra arg> does pass both the
+    # args in the attr as well as the <some extra arg> to the execution of the
+    # template
+    EXPECT_CONTAINS "$(bazel run examples/hellogrpc:staging.apply -- --v=1)" "apply --v=2 --v=1"
```

Comment:
Not sure if this is your test, but we have two —v on it

Reply:
I just copied how @nlopezgi did the e2e test in hellohttp